packages/server/src/cron/delete-old-builds.js

```diff
@@ -37,19 +49,42 @@ function startDeleteOldBuildsCron(storageMethod, options) {
     return;
   }
 
-  if (!options.deleteOldBuildsCron.schedule || !options.deleteOldBuildsCron.maxAgeInDays) {
-    throw new Error('Cannot configure schedule');
-  }
-
   /** @type {(s: string) => void} */
   const log =
     options.logLevel === 'silent'
       ? () => {}
       : msg => process.stdout.write(`${new Date().toISOString()} - ${msg}\n`);
 
+  /**
+   *
+   * @type {LHCI.ServerCommand.DeleteOldBuildsCron[]}
+   */
+  const cronConfig = Array.isArray(options.deleteOldBuildsCron)
+    ? options.deleteOldBuildsCron
+    : [options.deleteOldBuildsCron];
+
+  cronConfig.forEach((config, index) => {
+    if (!config.schedule || !config.maxAgeInDays) {
+      throw new Error(
+        `Can't configure schedule because you didn't specify 'schedule' field or 'maxAgeInDays' field in item with index: ${index +
```

Comment:
is there a middle ground between my suggestion and what you have here? :) 

I don't feel *very* strongly about my text, but definitely the context for `deleteOldBuilds` and using `index` instead of `index + 1` is important IMO.

Reply:
friendly ping, I'd love to get this merged for the imminent release

Reply:
Yea, I got what you mean. I didn't think about that 👍 I've replaced `index + 1` with `index`